Identifying who can use IP restrictions

runleonarun opened the pull request:
## What are you changing in this pull request and why?

Identifying the Enterprise tiers able to use ip restrictions.

## Checklist
<!--
Uncomment if you're publishing docs for a prerelease version of dbt (delete if not applicable):
- [ ] Add versioning components, as described in [Versioning Docs](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-entire-pages)
- [ ] Add a note to the prerelease version [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)
-->
- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `website/static/_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

runleonarun committed:
adding notes

runleonarun commented on website/docs/docs/cloud/manage-access/enterprise-permissions.md:
```diff
@@ -4,12 +4,7 @@ id: "enterprise-permissions"
-If you're interested in learning more about an Enterprise plan, contact us at sales@getdbt.com.
-
-:::
+<Snippet src="available-enterprise-only" />
```
Wonder if we should switch to partials:
`import SetUpPages from '/snippets/_available-enterprise-only.md';`

mirnawong1 replied:
addressed in https://github.com/dbt-labs/docs.getdbt.com/pull/3616

runleonarun commented on website/docs/docs/cloud/secure/ip-restrictions.md:
```diff
@@ -1,10 +1,12 @@
 sidebar_label: "IP restrictions"
 ---
 
+<Snippet src="available-tiers-iprestrictions" />
```
Switch to partials?
>`import SetUpPages from '/snippets/_available-tiers-iprestrictions.md';`

mirnawong1 replied:
addressed in https://github.com/dbt-labs/docs.getdbt.com/pull/3616

runleonarun commented on website/snippets/available-tiers-iprestrictions.md:
```diff
@@ -0,0 +1,7 @@
+:::info Limited to some dbt Cloud Enterprise tiers
```
If we switch to partials, rename this file to
> website/snippets/_available-tiers-iprestrictions.md

mirnawong1 replied:
addressed in https://github.com/dbt-labs/docs.getdbt.com/pull/3616

runleonarun commented on website/snippets/available-enterprise-only.md:
```diff
@@ -0,0 +1,5 @@
+:::info Limited to dbt Cloud Enterprise
```
If we switch to partials, rename this file to
> website/snippets/_available-enterprise-only.md

mirnawong1 replied:
addressed in https://github.com/dbt-labs/docs.getdbt.com/pull/3616

runleonarun commented on website/snippets/available-tiers-iprestrictions.md:
```diff
@@ -0,0 +1,7 @@
+:::info Limited to some dbt Cloud Enterprise tiers
```
```suggestion
:::info Limited to certain Enterprise tiers
```

runleonarun commented on website/snippets/available-enterprise-only.md:
```diff
@@ -0,0 +1,5 @@
+:::info Limited to dbt Cloud Enterprise
```
```suggestion
:::info Limited to Enterprise
```

runleonarun committed:
Apply suggestions from code review

mirnawong1 committed:
Merge branch 'current' into add-tier-note

mirnawong1 commented on website/snippets/available-tiers-iprestrictions.md:
```diff
@@ -0,0 +1,7 @@
+:::info Limited to certain Enterprise tiers
+
+IP restrictions can be configured in organizations using these dbt Cloud Enterprise tiers:
```
sugg to active voice
```suggestion
Organizations can configure IP restrictions using the following dbt Cloud Enterprise tiers:
```

mirnawong1 commented on website/snippets/available-tiers-iprestrictions.md:
```diff
@@ -0,0 +1,7 @@
+ * Business Critical 
+ * Virtual Private
+
+:::
```
```suggestion
To learn more about these tiers, contact us at <sales@getdbt.com>.

:::
```

mirnawong1 commented:
thank you @runleonarun , have two small suggestions and i also created a branch that addresses the partials file. You can merge it into this one if it's easier.

runleonarun committed:
Apply suggestions from code review

Co-authored-by: mirnawong1 <89008547+mirnawong1@users.noreply.github.com>

mirnawong1 committed:
change to partials (#3616)

change snippets to partial.md file per Leona's suggestion in pr #3612.
created branch to merge into `add-tier-note` branch for pr #3612

Co-authored-by: Leona B. Campbell <3880403+runleonarun@users.noreply.github.com>

mirnawong1 reviewed (approved):
lgtm @runleonarun !

JKarlavige commented on website/docs/docs/cloud/manage-access/enterprise-permissions.md:
```diff
@@ -4,14 +4,9 @@ id: "enterprise-permissions"
-If you're interested in learning more about an Enterprise plan, contact us at sales@getdbt.com.
-
-:::
-
-## Overview
+<SetUpPages features={'/snippets/_available-enterprise-only.md'}/>
```
Hey @runleonarun & @mirnawong1, just want to call out one thing here (left the same note in the [documentation pr](https://github.com/dbt-labs/docs.getdbt.com/pull/3624#pullrequestreview-1514768046)).

When using the partials, the file path does not have to be included when using the partial component. It can be written as:
```markdown
import SetUpPages from '/snippets/_available-enterprise-only.md';

<SetUpPages />
```

Props (the `features={...` part on the component) only need to be added if you want to pass in data from the component into the partial. This is useful for using the same partial component on multiple docs pages and displaying different values for each.

For example, if we wanted to use this partial on multiple pages and pass in a different 'feature' for each docs page, we can write it as:
```markdown
import SetUpPages from '/snippets/_available-enterprise-only.md';

<SetUpPages feature='A really cool feature' />
```

Then within the `/snippets/_available-enterprise-only.md` file, we can display that feature prop with:
```markdown
This feature: {props.feature}

other content etc...

// which translates to:

This feature: A really cool feature

other content etc...
```

That would allow us to use the same component on a different docs page, where we can show a separate feature, for example:
```markdown
import SetUpPages from '/snippets/_available-enterprise-only.md';

<SetUpPages feature='Another awesome feature' />
```

JKarlavige commented on website/docs/docs/cloud/secure/ip-restrictions.md:
```diff
@@ -1,11 +1,13 @@
 ---
 
-## About IP Restrictions
+import SetUpPages from '/snippets/_available-tiers-iprestrictions.md';
+
+<SetUpPages features={'/snippets/_available-tiers-iprestrictions.md'}/>
```
Same note for this, it can be used as:

```markdown
import SetUpPages from '/snippets/_available-tiers-iprestrictions.md';

<SetUpPages />
```
